heareval/predictions/task_predictions.py

```diff
@@ -30,8 +32,9 @@ def __init__(self, nfeatures: int, nlabels: int, prediction_type: str):
         super().__init__()
 
         self.projection = torch.nn.Linear(nfeatures, nlabels)
+        torch.nn.init.normal_(self.projection.weight)
```

Comment:
Was experimenting with different inits for testing -- can remove this.

Reply:
okay, I'm fine either way